tartufo/cli.py

```diff
@@ -158,34 +173,38 @@ def main(ctx: click.Context, **kwargs: config.OptionTypes) -> None:
     also be made to work in pre-commit mode, for scanning blobs of text as a
     pre-commit hook.
     """
+
     options = types.GlobalOptions(**kwargs)  # type: ignore
     ctx.obj = options
+    if options.quiet and options.verbose > 0:
+        raise click.BadParameter("-v/--verbose and -q/--quiet are mutually exclusive.")
```

Comment:
Oh hey, nice exception! I didn't know about this one! I wonder if there are other places we should be using this... (in other PRs, of course.)